pipeline/utils/helpers_test.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from pipeline.types import Match
+from pipeline.utils.helpers import find_match_of_3
+
+
+def test_3_match():
```

Comment:
Would be good to change the names of the test functions to be more descriptive. For example: `test_returns_match_of_three()`.